src/data/sponsors.ts

```diff
@@ -139,6 +138,22 @@ export const sponsors = [
       },
     ],
   },
+  {
+    sponsorLevelName: "Exhibitor",
+    sizeOnLargeScreen: 2,
+    sizeOnMediumScreen: 2,
+    sizeOnSmallScreen: 1,
+    showPopup: true,
+    list: [
+      {
+        name: "Things Cyber",
+        logoUrl: ThingsCyber.src,
+        url: "https://thingscyber.com.np/",
+        description:
+          "Things Cyber is a Kathmandu-based cybersecurity company offering AI-powered IT consulting, cloud services, and automated employee training to reduce security risks and enhance threat defense.",
+      },
+    ],
+  },
```

Comment:
This should be removed, as exhibitor package doesn't provide logo exposure on sponsor banner area but only at exhibitor info page.

> 
> Exhibitor package | 600 USD (or 50,000 NPR for Local sponsors)
> 
> Get a booth for your business in our exhibition area!
> Benets
> -    S size booth & Logistics support
> -    3 Tickets with exhibition area access only
> -    Logo exposure on exhibitors information page on our website
>